examples/protocol_specification_ex/contract_api.yaml

```diff
@@ -10,18 +10,18 @@ speech_acts:
     ledger_id: pt:str
```

Comment:
this is still a draft. Still need to understand what are the requirements to satisfy